.github/workflows/first_startup.yaml

```diff
@@ -0,0 +1,36 @@
+name: first startup
+on: [push, pull_request]
+jobs:
+
+  test:
+    name: Startup test
+    runs-on: ubuntu-latest
```

Comment:
Does it make sense to merge this with the osx startup tests and use the matrix to test both OSs?

Reply:
Then they're connected and you can't restart them individually ... we could ?

Reply:
You cannot restart them individually (I'd call that a bug of GH workflows UI), but you can at least have the failure of one not stop the other with `fail-fast: false` in `strategy:` (see e.g. `.github/workflows/integration.yaml` ).

Reply:
I am very divided on that, I preferred fail-fast: true for the connected jobs.

Reply:
On the other hand, we test Python-on-conda in the osx wf and migrations on Linux, so I'm fine keeping them separated.